controls/roles/validator-import-api/tasks/main.yml

```diff
@@ -80,3 +80,30 @@
     become: yes
     with_items: "{{ validator_keys }}"
   when: consClient == "nimbus"
+
+- block:
+  - name: Teku - Get validator service's api token
+    slurp:
+      src: "/opt/app/services/{{ beacon_service }}/data/validator/key-manager/validator-api-bearer"
```

Comment:
this should be the same as with prysm, because the path can change with user input

Reply:
can it be like nimbus or lighthouse? 
https://github.com/stereum-dev/ethereum-node/blob/main/controls/roles/validator-import-api/tasks/main.yml#L43

Reply:
sure, if it works :)

Reply:
done!